avm/res/app/session-pool/main.bicep

```diff
@@ -0,0 +1,273 @@
+metadata name = 'Container App Session Pool'
+metadata description = 'This module deploys a Container App Session Pool.'
+metadata owner = 'Azure/module-maintainers'
```

Comment:
We recently removed this metadata from all modules, due to the challenge of maintaining the same and the small value added by that. The module ownership is anyway centrally maintained in the AVM website.

Please remove this line and rerun the Set-AVMModule utility to reflect the change
```suggestion
```